### PR TITLE
Session5

### PR DESCRIPTION
**Pull request by emorejejeh**

### CODE REVIEW

File: ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        [HttpGet("{itemId}")]
+        public IActionResult Get(int itemId)
+        {
+            return Ok(_itemService.GetById(itemId));
+        }
+
+        [HttpGet("filter")]
+        public IActionResult GetByFilters([FromQuery] Dictionary<string, string> filters)
+        {
+            return null;
```
Comment (kevinatgithub) [inline ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs]: Please implement method, Thank you!

File: ItemService/ItemService.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
```
Comment (kevinatgithub) [inline ItemService/ItemService.cs]: Can you please give more info for the purpose of this file? Thank you!

File: ItemService/ItemService.csproj [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (kevinatgithub) [inline ItemService/ItemService.csproj]: Can you please give more details for the purpose of this project? Thank you!

File: Services/ItemService.cs [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        }
+
+        public IEnumerable<ItemDto> GetAll()
+        {
+            return _itemRepository.All();
+        }
+
+        public IEnumerable<ItemDto> GetAllByFilter(ItemFilterByDto filters)
+        {
+            return null;
```
Comment (kevinatgithub) [inline Services/ItemService.cs]: Please implement method, Thank you!

**Review by kevinatgithub — CHANGES_REQUESTED** [review]

File: ItemService/ItemService.cs (hunk 2 above)
Comment (emorejejeh) [reply ItemService/ItemService.cs]: removed

File: Services/ItemService.cs (hunk 4 above)
Comment (emorejejeh) [reply Services/ItemService.cs]: implemented

File: ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs (hunk 1 above)
Comment (emorejejeh) [reply ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs]: removed

**Review by jegtugado — APPROVED** [review]

**Pull request merged**